content/workshops/pki_workshop/unit_lessons/unit_11.md

```diff
@@ -0,0 +1,80 @@
+# Unit 11: Certificate profiles
+
+In this unit you will go over the caUserCert profile. This certificate profile is for enrolling user certificates.
+
+The CA profiles are located here:
```

Comment:
CA should be defined on first use in each unit.

Reply:
Fixed